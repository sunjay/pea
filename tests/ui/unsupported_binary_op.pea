fn main() {
    println!(b"3245" - 3);
}
