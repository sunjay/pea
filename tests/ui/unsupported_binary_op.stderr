stack backtrace:
   0: main
error: unsupported binary operator `-` for types `[u8]` and `i64`
